packages/buckets/src/api/index.ts

```diff
@@ -505,7 +529,7 @@ export async function bucketsPushPathNode(
   input: any,
   opts?: PushOptions,
   ctx?: ContextInterface,
-) {
+): Promise<PushPathResult> {
```

Comment:
Re: Above, our linter will now try to enforce explicitly defined types on external APIs.